### PR TITLE
I18n sync Down & Out 05/21

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: pegasus/cache/i18n/id-ID.yml [hunk 1]
```diff
@@ -2191,11 +2191,3 @@
     Account” option:'
   hero_name: 'Minecraft: Hero''s Journey'
   applab-intro_name: App Lab Intro
-  homepage_below_hero_announcement_text: We care about your privacy and are making
```
Comment (tanyaparker) [inline pegasus/cache/i18n/id-ID.yml:2194]: How come these strings are being removed from the pegasus files?

Comment (Hamms) [reply pegasus/cache/i18n/id-ID.yml:2194]: Huh, I'm not sure

Comment (Hamms) [reply pegasus/cache/i18n/id-ID.yml:2194]: Oh, likely because we need another sync up. I'll get on it

Comment (Hamms) [reply pegasus/cache/i18n/id-ID.yml:2194]: closing in favor of https://github.com/code-dot-org/code-dot-org/pull/22600

**Pull request closed without merging**